cmd/kubeadm/app/cmd/join.go

```diff
@@ -18,20 +18,12 @@ package cmd
 
 import (
 	"fmt"
+
```

Comment:
```suggestion
```

please keep the commits squashed to one.
i didn't find other problems but i might have missed them.

Reply:
done